src/modules/client/components/planning/EventCreationModal.vue

```diff
@@ -179,6 +183,14 @@ export default {
       }
       this.$emit('update:newEvent', payload);
     },
+    toggleAddressSelect () {
+      const addressList = this.customerAddressList(this.newEvent);
+      const addressIndex = addressList.findIndex(ev => this.newEvent.address.label === ev.label);
+      // eslint-disable-next-line prefer-destructuring
+      if (addressIndex === 0) this.newEvent.address = addressList[1];
+      // eslint-disable-next-line prefer-destructuring
+      else this.newEvent.address = addressList[0];
```

Comment:
destructuring : 
[ this.newEvent.address ] = addressList;
this.newEvent.address = addressList[1];
https://eslint.org/docs/rules/prefer-destructuring

En soit on peut supprimer la règle pour les array, on l'utilise plutôt pour les objets.

Reply:
Vous en pensez quoi ? Je n'ai pas l'impression qu'on utilisait trop cette syntaxe et je la trouve un peu moins claire.

Reply:
Perso je suis d'accord, je n'aime pas le destructuring et j'ai pas trouvé grand chose pour justifier son existence